docs/customization.md

```diff
@@ -11,7 +11,7 @@ _This can be used to verify that the provided email is not a disposable one._
 
 ```ts
 authenticator.use(
-  new OTPStrategy({
+  new TOTPStrategy({
```

Comment:
Thanks for this one! The library has undergone a major re-write, and there may be small remnants from that.